queries/ralph/highlights.scm

```diff
@@ -0,0 +1,135 @@
+(type_identifier) @type
+
+[
+  (primitive_type)
+  (alph_token_id)
+] @type.builtin
+
+[
+  (var_declaration)
+  (struct_field)
+] @variable
+
+(arg_def
+  (identifier) @variable)
+
+(struct_constructor_field
+  (identifier) @variable)
```

Comment:
Seems like sometimes variables in random places (like `key` in `let depositor = key`) aren't highlighted as `@variable`; would it be easy to place (at the top of the file) just: `(identifier) @variable` and then override the rest (like `(func_id (identifier) @function)`) later? Genuinely asking. If there are too many other non-variable identifier nodes then it's fine

Reply:
I gave a try, it looks good with that.

Reply:
Hopefully I made everything well this time :)

Reply:
Thanks!